### PR TITLE
TE-3361 Add empty line before each return expression

### PR DESCRIPTION
**Pull request by stereomon**
- Developer(s): @stereomon

- Ticket: https://spryker.atlassian.net/browse/TE-1210

- Academy PR: ACADEMY_URL_HERE

- Release Group: URL_HERE

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   CodeSniffer           | patch (currently 0.x)  |                       |

#### Release Notes

Added new rules from slevomat which add an empty line before control structures.

-----------------------------------------

### Improvement

- Added new rules from slevomat which add an empty line before control structures.

### CODE REVIEW

File: Spryker/ruleset.xml [hunk 1]
```diff
@@ -54,6 +54,15 @@
     <rule ref="SlevomatCodingStandard.Commenting.EmptyComment"/>
     <rule ref="SlevomatCodingStandard.Commenting.DisallowOneLinePropertyDocComment"/>
 
+    <rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing.IncorrectLinesCountBeforeControlStructure"/>
```
Comment (dereuromark) [inline Spryker/ruleset.xml]: why is this needed?

Comment (stereomon) [reply Spryker/ruleset.xml]: To add an empty line before each return expression. When you have something in your mind why you think it's not required then please tell me and don't play comment ping pong.

We have different code style because of this missing rule. Sometimes we have an empty line before a return expression and sometimes not. This fixes this issue. 

Instead of

```
$foo = 'bar';
return $foo;
```

we will have

```
$foo = 'bar';

return $foo;
```

And this makes the code look better.

Comment (dereuromark) [reply Spryker/ruleset.xml]: the line after this should already suffice. i dont see how this helps here.

Comment (dereuromark) [reply Spryker/ruleset.xml]: Ah, this is a way to boolean enable this check?

**Comment by dereuromark** [comment]
Please also run
composer docs

**Comment by dereuromark** [comment]
Wlll be merged FR 17th.
